Dockerfile.puppeteer

```diff
@@ -0,0 +1,41 @@
+FROM node:8-slim
+
+# See https://crbug.com/795759
+RUN apt-get update && apt-get install -yq libgconf-2-4
```

Comment:
Please move this into the lower `RUN` part, so that the apt-cache is not part of the image.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run